### PR TITLE
chore: added cluster-pool configuration to hub yaml

### PR DESCRIPTION
**Pull request by r2dedios**
PR for adding the ClusterPool creation needed by Resilient Cluster Addon

### CODE REVIEW

File: values-hub.yaml [hunk 1]
```diff
@@ -154,6 +154,44 @@ clusterGroup:
...
+        provider:
+          region: ca-central-1
+        platform:
+          aws:
+            region: ca-central-1
+        size: 1
+        baseDomain: fsi-partner.rhecoeng.com
+        runningCount: 0
+        cluster:
+          install-config: "ESTO ES UNA PRUEBA"
```
Comment (TomerFi) [inline values-hub.yaml:193]: Perhaps we should add some info about the target install config in a comment here.

**Review by TomerFi — CHANGES_REQUESTED** [review]

File: common/acm/templates/provision/clusterpool.yaml [hunk 2]
```diff
@@ -54,7 +54,7 @@ spec:
   runningCount: {{ $numClusters }}
   baseDomain: {{ .baseDomain }}
   installConfigSecretTemplateRef:
-    name: {{ $poolName }}-install-config 
+    name: {{ $poolName }}-install-config
```
[the remark this answers is not in the document]
Comment (r2dedios) [reply common/acm/templates/provision/clusterpool.yaml:57]: you're right, It was my editor. Removing it

File: values-hub.yaml (hunk 1 above)
Comment (r2dedios) [reply values-hub.yaml]: oh sorry, it was a comment for testing the install-config creationg. Removed.

**Comment by r2dedios** [comment]
Updated changes of our last talk @TomerFi

**Review by TomerFi — APPROVED** [review]
LGTM

Merging.

**Pull request merged**